qt/scientific_interfaces/Inelastic/Analysis/IIndirectFitDataView.h

```diff
@@ -42,6 +43,7 @@ class MANTIDQT_INELASTIC_DLL IIndirectFitDataView {
   virtual void clearTable() = 0;
   virtual QString getText(int row, int column) const = 0;
   virtual QModelIndexList getSelectedIndexes() const = 0;
+  virtual bool DataColumnContainsText(QString text) const = 0;
```

Comment:
This is very picky, but can the first letter in this function name be lower case?

Reply:
Can you also change the parameter to be a `std::string const &`? Currently the `text` parameter is being passed in by value, meaning each time a parameter is passed in a copy is made https://www.ibm.com/docs/en/zos/2.4.0?topic=calls-pass-by-reference-c-only

Reply:
We are going to erradicate all QString from Mantid. :)